test/pkg/integration/setup.go

```diff
@@ -209,39 +210,43 @@ func waitOnGitpodRunning(namespace string, waitTimeout time.Duration) env.Func {
 }
 
 func logGitpodStatus(t *testing.T, client klient.Client, namespace string) {
-	t.Logf("Gitpod components status:")
 	var allPods corev1.PodList
 	err := client.Resources(namespace).List(context.Background(), &allPods)
 	if err != nil {
-		t.Logf("failed to list pods: %v", err)
+		t.Logf("failed to list pods to log gitpod status: %v", err)
 		return
 	}
 
+	var buf strings.Builder
+	tw := tabwriter.NewWriter(&buf, 0, 0, 1, ' ', 0)
+	_, _ = tw.Write([]byte("Component\tPod\tReady\tStatus\tRestarts\tAge\n"))
+
```

Comment:
using a tabwriter makes it easier to read the logs by having all items aligned, see e.g. https://github.com/gitpod-io/gitpod/actions/runs/5423326857/jobs/9861495589#step:8:1646